<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSearchDialog</class>
 <widget class="QDialog" name="NodeSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search for nodes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Trees</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="treeSelected">
        <property name="text">
         <string>Selected</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="treeVisible">
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="treeAll">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="nodeType">
        <property name="text">
         <string>Node type</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QCheckBox" name="typePresyn">
           <property name="toolTip">
            <string>A presynaptic terminal on the target tree</string>
           </property>
           <property name="text">
            <string>Presynaptic terminal</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="typePostsyn">
           <property name="toolTip">
            <string>A postsynaptic density on the target tree</string>
           </property>
           <property name="text">
            <string>Postsynaptic density</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="typePrePartner">
           <property name="toolTip">
            <string>A presynaptic terminal on any tree that forms a synapse with a target tree</string>
           </property>
           <property name="text">
            <string>Presynaptic partner</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="typePostPartner">
           <property name="toolTip">
            <string>A postsynaptic density on any tree that forms a synapse with a target tree</string>
           </property>
           <property name="text">
            <string>Postsynaptic partner</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="typeBranchpoint">
           <property name="toolTip">
            <string>A node where several branches join</string>
           </property>
           <property name="text">
            <string>Branch point</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="typeEndpoint">
           <property name="toolTip">
            <string>A node at the end of a branch (but not a synaptic node)</string>
           </property>
           <property name="text">
            <string>Endpoint</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="typeExit">
           <property name="toolTip">
            <string>The node where a cell's main process leaves the neuropil</string>
           </property>
           <property name="text">
            <string>Neuropil exit point</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="typeSoma">
           <property name="toolTip">
            <string>The cell body</string>
           </property>
           <property name="text">
            <string>Soma</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="memo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Memo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <widget class="QRadioButton" name="memoContains">
           <property name="text">
            <string>Contains</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="memoText"/>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="memoEquals">
           <property name="text">
            <string>Equals</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="memoAndAlso">
           <property name="text">
            <string>and also</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="memoAndText">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="memoButNot">
           <property name="text">
            <string>but not</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="memoNotText">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="count">
        <property name="text">
         <string>–</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NodeSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>938</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NodeSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>944</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memo</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>723</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeType</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoContains</sender>
   <signal>toggled(bool)</signal>
   <receiver>memoAndAlso</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>788</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>828</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoContains</sender>
   <signal>toggled(bool)</signal>
   <receiver>memoButNot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>866</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>typePrePartner</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>typePostPartner</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoAndAlso</sender>
   <signal>toggled(bool)</signal>
   <receiver>memoAndText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>817</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>812</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoButNot</sender>
   <signal>toggled(bool)</signal>
   <receiver>memoNotText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>852</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
